### ISSUE TITLE
Shall we add genus to 8.2.12.2?

### ISSUE THREAD

**Issue by robinwhittleton** [standardebooks/manual#128]
At the moment, 8.2.12.2 says ”Family, order, class, phylum or division, and kingdom names are capitalized but not italicized.” Shall we add genus and tribe to that?

(This arose from a question from a new producer who found an italicised reference to a monomial genus https://en.wikipedia.org/wiki/Coleogyne.)

**Comment by acabal** [comment]
Yes, I suppose so. I think genus was excluded in the manual because in texts from the PD time period it wasn't usually capitalized. We would have to go back through the corpus and update things. There are quite a few instances where genus is not capitalized, and also quite a few instances where we'd have to look up if to see if it's referring to a genus or species. Not sure if you're interested in doing that. A rough search found 124 matches that need review.

`se xpath "//*[contains(@epub:type, 'z3998:taxonomy') and re:test(., '^[a-z]')] /path/to/ebooks`

**Comment by robinwhittleton** [comment]
Haha, yeah, dropped myself in that one. OK, I’ll get these fixed.

**Comment by robinwhittleton** [comment]
In this sentence, should we recommend wrapping the parts with `<span epub:type="z3998:taxonomy">` too?

```
<p>A bonobo monkey is in the phylum Chordata, class Mammalia, order Primates.</p>
```

The spec for `taxonomy` suggests “the name of a species, genus or other taxonomic classification.”

**Comment by robinwhittleton** [comment]
Hmm, also: do we want to recommend `xml:lang="la"` for taxonomy too? There’s a mix in the corpus, most don’t.

**Comment by acabal** [comment]
> In this sentence, should we recommend wrapping the parts with `<span epub:type="z3998:taxonomy">` too?
> 
> ```
> <p>A bonobo monkey is in the phylum Chordata, class Mammalia, order Primates.</p>
> ```
> 
> The spec for `taxonomy` suggests “the name of a species, genus or other taxonomic classification.”

Eh, that's going down the rabbit hole of tagging things that wouldn't otherwise get tags. The general philosophy is that if it has a tag anyway, like `<i>`, then we may as well give it semantics. But adding tags just for the sake of semantics is optional, though accepted.


> Hmm, also: do we want to recommend `xml:lang="la"` for taxonomy too? There’s a mix in the corpus, most don’t.

I would say no because taxonomies are more like proper names, and I don't think we pronounce them today in the way that Latin was spoken anyway.
